index.js

```diff
@@ -1,27 +1,23 @@
-
-
-function menu(){
-  let nav= document.getElementById("nav-header")  
-    nav.style.display="flex"
-    let icon= document.getElementById('close-icon')
-    let burger=document.getElementById('burger')
-    burger.style.display="none"
-    icon.style.display="inline-block"
-    document.getElementById('img-deco1').style.display="block"
-    document.getElementById('img-dots').style.display="block"
-    document.getElementById('img-deco2').style.display="block"
-
+function menu() {
+  const nav = document.getElementById('nav-header');
+  nav.style.display = 'flex';
+  const icon = document.getElementById('close-icon');
+  const burger = document.getElementById('burger');
+  burger.style.display = 'none';
+  icon.style.display = 'inline-block';
+  document.getElementById('img-deco1').style.display = 'block';
+  document.getElementById('img-dots').style.display = 'block';
+  document.getElementById('img-deco2').style.display = 'block';
```

Comment:
- [ ] [OPTIONAL] After toggling the menu in the mobile menu, it affects the desktop version as it hides the nav items and displays the hamburger on the desktop version. I highly recommend you fix this as it will help improve your coding skills.
![Capture](https://user-images.githubusercontent.com/85095702/189905776-206d270c-6f76-4ec8-9fcd-eda12b67b594.PNG)